PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-azure-sdk/src/main/java/com/microsoft/azure/toolkit/intellij/azure/sdk/buildtool/ServiceBusReceiverAsyncClientCheck.java

```diff
@@ -0,0 +1,34 @@
+package com.microsoft.azure.toolkit.intellij.azure.sdk.buildtool;
+
+import com.intellij.codeInspection.*;
+import com.intellij.psi.*;
```

Comment:
nit: we usually don't collapse namespaces (but IntelliJ does 🤐), you can disable it, there are some tips here https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports